src/lib.rs

```diff
@@ -499,7 +544,18 @@ impl<'h, 'b> Response<'h, 'b> {
         complete!(skip_empty_lines(&mut bytes));
         self.version = Some(complete!(parse_version(&mut bytes)));
         space!(bytes or Error::Version);
```

Comment:
Somewhat off-topic for this PR, but I was tempted to change this to return `Error::Status` instead. Without the new flag enabled, `HTTP/1.1   200\r\n` currently will fail with `Error::Version` which doesn't seem quite right. I left it alone to minimize this change though.

Reply:
I... do not why Github is not rendering the spaces in that markdown correctly. How about in block form:
```
HTTP/1.1   200\r\n
```